Demo/Storybook/Storybook/Component.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Component.swift
+//  Storybook
+//
+//  Created by Borut Tomazin on 23/01/2023.
+//
+
+import Foundation
+
+enum Component: String, Identifiable, Hashable, CaseIterable {
```

Comment:
Based on the usage we can omit conformance to all except `CaseIterable`.

on `ContentView` we can change the ForEach method to the following 
```swift
ForEach(Component.allCases, id: \.self)
```

Reply:
True. Thanks 🙌